api/src/main/java/org/apache/cloudstack/api/command/user/vm/DeployVMCmd.java

```diff
@@ -373,6 +381,13 @@ public Long getZoneId() {
     }
 
     public List<Long> getNetworkIds() {
+        if (MapUtils.isNotEmpty(vAppNetworks)) {
```

Comment:
why is this condition repeated in the private method? is there a chance it has changed?

Reply:
I was not sure if the private method will be used only in this section. It won't be changed. Will refactor it once we have sync with Paul on mapping @DaanHoogland